packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -58,6 +59,11 @@ const Color _borderColor = CupertinoDynamicColor.withBrightness(
   darkColor: Color(0xFF57585A),
 );
 
+const Color _kBackgroundColor = CupertinoDynamicColor.withBrightness(
```

Comment:
This is also defined in CupertinoContextMenuAction: https://github.com/flutter/flutter/blob/418fa9d62610b6990f18c0a7cf6c194364fad4b8/packages/flutter/lib/src/cupertino/context_menu_action.dart#L50

How can we remove the duplicate/use one for both?

Reply:
We should probably expose these in one, then use them in the other? As in CupertinoContextMenuAction can use `CupertinoContextMenu.kBackgroundColor`. The `actions` property is asking for any kind of widget, so providing these constants so somebody could make their own, slightly different CupertinoContextMenuAction that still uses some constants from us makes sense.

If it makes sense to either move these to the constants file or use an existing constant we can, but if the styling is specific to these widgets I think it's fine to leave them in these files. It improves specific discoverability in the API docs that way.